src/intl/en/common.json

```diff
@@ -113,6 +113,7 @@
   "learn-hub": "Learn Hub",
   "learn-menu": "Learn menu",
   "learn-more": "Learn more",
+  "learning-quizzes": "Learn quizzes",
```

Comment:
we're not using this string

```suggestion
```

Reply:
Ill change the string to match what we are doing now around scrapping learning, and just call it quizzes. 

But, this is used by breadcrumbs, so we do need this in there.